scss/edittoolbar.scss

```diff
@@ -0,0 +1,36 @@
+// Plone Barceloneta Styling & components
+
+// Plone colors
+@import "variables.colors.plone";
+@import "variables.colors.dark.plone";
+
+// Plone specific default variables & properties
+@import "variables.properties";
+@import "variables.barceloneta";
+
+@import "base";
```

Comment:
You import "base" here ... that generates a lot of stuff, which isn't really needed for the toolbar (but the title says so). At least I'm not sure about the purpose of "edittoolbar.css" ... can you elaborate a usecase for this?

Reply:
I think this imports can be a starting point to make the toolbar works without base:
```scss
@import "variables.colors.plone";
@import "variables.colors.dark.plone";
@import "root_variables";
@import "toolbar";
@import "scaffolding";
```

Reply:
The toolbar depends on bootstrap dropdowns which are missing in your case. I'm really not sure, because the linked issue says "render toolbar **and** the editforms" with barceloneta. We could try to extract this in a "backend.css" but I doubt that this would be much smaller than the complete barceloneta package now, because we have dropdowns, forms, buttons, grids, etc. in the backend.

We ususally depend on `@plone/barceloneta-theme-base` in our themes and decide what whe need by adjusting the imports like its done in the bootstrap example here https://getbootstrap.com/docs/5.3/customize/sass/#importing (Options B)

Reply:
I would say "for edit forms, use Barceloneta using Diazo `backend.xml`" (*). @erral did you mean "support edit forms in a Diazo Theme" or it was just a reference? I mean, style edit forms in your theme without Barceloneta can take time, the same amount to make the toolbar work in your theme directly.

(*) Plone 6.1 Classic UI supports `backend.xml` and Plone 6.1 introduces the nice `portal_url` variable by default in the theme advanced control panel that can be used in Diazo to anchor resources to the root.

Reply:
An Idea: If Bootstrap is already available in your theme, the toolbar should be usable by including only the compiled `scss/_toolbar.scss` ... if we get rid of the `--#{$prefix}` syntax there (simply use `--bs`) this should be compileable as a standalone css file.